cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/template/etl/batch/sink/BatchElasticsearchSink.java

```diff
@@ -50,7 +50,7 @@
 @Description("CDAP Elasticsearch Batch Sink takes the structured record from the input source" +
   " and converts it to a json, then indexes it in elasticsearch using the index, type, and id specified by the user." +
   "The elasticsearch server should be running prior to creating the adapter.")
-public class ElasticsearchSink extends BatchSink<StructuredRecord, Writable, Writable> {
+public class BatchElasticsearchSink extends BatchSink<StructuredRecord, Writable, Writable> {
   private static final String INDEX_DESC = "The name of the index where the data will be stored. " +
     "If the index does not already exist, it will be created using elasticsearch's default properties";
```

Comment:
Add trailing periods to these descriptions. The variable names should be consistent with others and end with "_DESCRIPTION".